flang/test/Lower/pointer.f90

```diff
@@ -0,0 +1,16 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
+
+! CHECK-LABEL: func @_QPpointertests
+subroutine pointerTests
+  ! CHECK: fir.global @_QFpointertestsEptr1 : !fir.ptr<i32>
+  integer, pointer :: ptr1 => NULL()
```

Comment:
Could you expand these tests a bit so that it actually checks the NULL init, currently they show that the pointer variable type lowering and name mangling are OK, but not that their data pointers are set to zero.